src/models/loginModel.js

```diff
@@ -0,0 +1,23 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const userSchema = new Schema({
+
+  email: {
+    type: String,
+    required: [true, 'please input an email'],
+    unique: true,
+    match: /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/,
+  },
+  password: {
+    type: String,
+    required: [true, 'please input a password'],
+    minlength: 6,
+  },
+
+});
+
+const User = mongoose.model('users', userSchema);
+
+export default User;
```

Comment:
Just like on the Register feature, call this a user model, not the action you wanna take on it like register or login. It's essentially duplicate code we're talking about here.